platforms/shared/charts/vault-k8s-mgmt/values.yaml

```diff
@@ -58,3 +52,14 @@ k8s:
   #Provide the kubernetes host url
   #Eg. kubernetes_url: https://10.3.8.5:6443
   kubernetes_url: 
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
```

Comment:
Update the chart.yaml with new version and Readme

Reply:
Done